app/src/online/java/com/kamron/pogoiv/updater/AppUpdateUtil.java

```diff
@@ -94,4 +103,15 @@ public void onClick(DialogInterface dialogInterface, int i) {
         AlertDialog dialog = builder.create();
         return dialog;
     }
+
+    public static void deletePreviousApkFile(Context context) {
+        if (ContextCompat.checkSelfPermission(context,
+                Manifest.permission.WRITE_EXTERNAL_STORAGE) == PackageManager.PERMISSION_GRANTED){
+            String newApkFilePath = context.getExternalFilesDir(Environment.DIRECTORY_DOWNLOADS) + "/" + FILE_NAME;
```

Comment:
Re-read some of the comments. At least @DJCrapsody was reporting that a copy of the APK was ending up in the Downloads folder. So, this seems okay. What about the update.apk that people were talking about? Is that why you call this a "partial" fix?